### PR TITLE
feature (filter-toolbar): add missing tooltips and creating module for tooltip directive

### PR DESCRIPTION
**Pull request by adumitrescu-plenty**
+ add TooltipDirective to module and to publicApi

@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Team plentyShop about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

**Review by OliverBhm — DISMISSED** [review]

File: src/lib/public-api.ts [hunk 1]
```diff
@@ -184,6 +184,7 @@ export { SelectSortPipe } from './pipes/select-sort.pipe';
 export { AllowedColors } from './components/forms/select-box/data/allowed.colors.enum';
 export { SelectSortDirective } from './components/forms/select-box/directive/select-sort.directive';
 export { TerraRouteIcon } from './types/terra-route-icon.type';
+export { TooltipModule } from './components/tooltip/tooltip.module';
```
Comment (pweyrich) [inline src/lib/public-api.ts:187]: i am not sure if we should add this to the public api

**Review by Neill83 — APPROVED** [review]

**Review by Neill83 — CHANGES_REQUESTED** [review]
add a changelog

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes
 
 -   **terra-node-tree** add support for mat-icons.
+-   **TooltipDirective** added its own module - **TooltipModule**.
```
Comment (Neill83) [inline CHANGELOG.md:6]:
```suggestion
-   **tcTooltip** added its own module - **TooltipModule**.
```

**Review by Neill83 — CHANGES_REQUESTED** [review]

File: src/lib/components/tooltip/tooltip.module.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { NgModule } from '@angular/core';
+import { TooltipDirective } from './tooltip.directive';
+
+@NgModule({
+    imports: [],
+    declarations: [TooltipDirective],
+    exports: [TooltipDirective]
+})
+export class TooltipModule {}
```
Comment (pweyrich) [inline src/lib/components/tooltip/tooltip.module.ts:9]:
```suggestion
export class TerraTooltipModule {}
```
In order to be consistent with the other exported modules

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes
 
 -   **terra-node-tree** add support for mat-icons.
+-   **tcTooltip** added its own module - **TooltipModule**.
```
Comment (pweyrich) [inline CHANGELOG.md:6]:
```suggestion
-   **tooltip** added `TerraTooltipModule` to be able to individually include tooltip functionality.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes
 
 -   **terra-node-tree** add support for mat-icons.
+-   **tooltip** added `TerraTooltipModule` to be able to individually include tooltip functionality.
```
Comment (Neill83) [inline CHANGELOG.md:6]:
```suggestion
-   **tcTooltip** added `TerraTooltipModule` to be able to individually include tooltip functionality.
```

Comment (Neill83) [reply CHANGELOG.md:6]: it should be tcTooltip since its the selector of the tooltip and we always use the selector in there somehow...

**Review by Neill83 — CHANGES_REQUESTED** [review]

**Review by Neill83 — APPROVED** [review]

**Review by niwolf — APPROVED** [review]

**Review by nappynapster — APPROVED** [review]

**Pull request merged**